core/server/worker/src/main/java/alluxio/worker/AlluxioWorkerProcess.java

```diff
@@ -117,7 +117,7 @@
       // registered at worker registry, so the maximum timeout here is set to the multiply of
       // the number of factories by the default timeout of getting a worker from the registry.
       CommonUtils.invokeAll(callables,
-          (long) callables.size() * Constants.DEFAULT_REGISTRY_GET_TIMEOUT_MS);
+          (long) callables.size() * 10 * Constants.DEFAULT_REGISTRY_GET_TIMEOUT_MS);
```

Comment:
`DEFAULT_REGISTRY_GET_TIMEOUT_MS` is used for default registry, add 10 * instead of change the default value of constant.